utils.zsh/toolchain/windows-cross-toolchain.cmake

```diff
@@ -6,8 +6,8 @@ elseif(CMAKE_SYSTEM_PROCESSOR STREQUAL "x86")
     set(_CROSS_PREFIX "i686")
 endif()
 
-set(CMAKE_C_COMPILER ${_CROSS_PREFIX}-w64-mingw32-gcc)
-set(CMAKE_CXX_COMPILER ${_CROSS_PREFIX}-w64-mingw32-g++)
+set(CMAKE_C_COMPILER ${_CROSS_PREFIX}-w64-mingw32-gcc-posix)
```

Comment:
Instead of changing this globally, create a new file `windows-cross-toolchain-posix.cmake` with them.

Reply:
@PatTheMav Done. Does that look right what I did?

Reply:
Looks like this has been addressed, @PatTheMav ?